src/renderer/actions/network.ts

```diff
@@ -50,9 +51,7 @@ export const saveSelectedNetworkAction = (network: string): ISaveSelectedNetwork
 export const saveBeaconNodeAction = (url: string, network?: string) => {
     return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
         const localDockerName = network ? BeaconChain.getContainerName(network) : undefined;
-        const signingKey = PrivateKey.fromBytes(
-            Buffer.from(getState().register.signingKey.replace("0x",""), "hex")
-        );
+        const signingKey = PrivateKey.fromBytes(fromHex(getState().register.signingKey.replace("0x","")));
```

Comment:
isn't from hex already removing 0x?

Reply:
No, that's just to create Buffer from string. We don't have much usage of this currently

Reply:
Can you add 0x removal there so we don't copy code everywhere